package.json

```diff
@@ -12,11 +12,7 @@
     "watch": "gulp watch",
     "test": "gulp test",
     "tdd": "gulp tdd",
-    "changelog": "gulp changelog",
-    "release": "gulp release:minor",
-    "release:patch": "gulp release:patch",
-    "release:minor": "gulp release:minor",
-    "release:major": "gulp release:major"
+    "changelog": "gulp changelog"
```

Comment:
Should we also update the 3 urls that start with "https://github.com/mjeanroy/rollup-plugin-license"?

Reply:
Changed all the urls in the package.json